.travis.yml

```diff
@@ -0,0 +1,33 @@
+sudo: required
+
+language: scala
+
+services:
+- docker
+
+cache:
+  directories:
+  - $HOME/.ivy2/cache
+  - $HOME/.sbt/boot/
+
+script:
+- mkdir -p ${HOME}/.sbt
+- docker run -v ${HOME}/.ivy2:/root/.ivy2 -v ${HOME}/.sbt:/root/.sbt -v ${PWD}:/usace-flood-geoprocessing
```

Comment:
Do we have to use docker? Or can we get away with just raw `./sbt "project server" compile`?